lib/maze/client/appium/base_client.rb

```diff
@@ -23,11 +23,16 @@ def start_session
                                  Maze.driver.session_capabilities['appPackage']
                                when 'ios'
                                  unless app_id = Maze.driver.session_capabilities['CFBundleIdentifier']
-                                    app_id = Maze.driver.session_capabilities['bundleID']
+                                    app_id = Maze.driver.session_capabilities['bundleId']
```

Comment:
I understand we started seeing the issue with Appium 2.5 - does this fix work before and after 2.5 on BB and also on BS?  I don't think CI covers that.

Reply:
I've checked BB and can confirm that it works for Appium versions lower than 2.5.

Reply:
I've also checked BS and its working on Appium 1.21.0